.eslintrc.json

```diff
@@ -101,6 +99,48 @@
           }
         ]
       }
+    },
+    // rules which apply only to Markdown
+    {
+      "files": [
+        "**/*.{md,mdx}"
+      ],
+      "extends": [
+        "plugin:mdx/recommended"
+      ],
+      "settings": {
+        "mdx/code-blocks": true
+      }
+    },
+    {
+      "files": [
+        "**/*.{md,mdx}",
+        "**/*.{md,mdx}/**"
+      ],
+      "parserOptions": {
+        "project": false
+      },
+      "rules": {
+        "camelcase": 0,
+        "no-constant-condition": 0,
+        "no-console": 0,
+        "no-empty-pattern": 0,
+        "no-unused-vars": 0,
+        "no-undef": 0,
+        "react/no-unescaped-entities": 0,
+        "react/react-in-jsx-scope": 0,
+        "react/jsx-no-undef": 0,
+        "react/jsx-key": 0,
+        "react/jsx-no-comment-textnodes": 0,
+        "import/no-anonymous-default-export": 0,
+        "prettier/prettier": 0,
+        "jsx-a11y/anchor-is-valid": 0,
+        "jsx-a11y/accessible-emoji": 0,
+        "jsx-a11y/label-has-for": 0,
```

Comment:
a11y rules should be re-enabled over time, however I didn't want to introduce changes to the code examples other than fixing actual issues.